InternetHospital.WebApi/Controllers/AppointmentsController.cs

```diff
@@ -47,9 +53,9 @@ public IActionResult AddAppointment([FromBody] AppointmentCreationModel creation
                 return BadRequest(new { message = "Wrong claims" });
             }
 
-            var (status, message) = _appointmentService.AddAppointment(creationModel, doctorId);
+            var status = _appointmentService.AddAppointment(creationModel, doctorId);
 
-            return status ? (IActionResult)Ok(new { message }) : BadRequest(new { message });
+            return status ? (IActionResult)Ok(new { message = "Ok" }) : BadRequest();
```

Comment:
Is it necessary to return message?

Reply:
actually no, but front end still waiting for a message. Without it we gonna have an error on client side. So i preferred to leave this temporary message instead of creation another pull request on UI.